small/chmod/chmod.c

```diff
@@ -0,0 +1,10 @@
+#include <sys/stat.h>
+#include <stdio.h>
+#include <errno.h>
+
+int main() {
+	int rc = chmod("chmod_test_dir", 0700);
+	if (rc < 0)
+		printf("chmod failed errno=%d\n", errno);
```

Comment:
Should return early / exit the program with failure e.g. exit(EXIT_FAILURE).

Thank you.